roles/ideascube/files/ideascube_leftover_files.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Run ideascube clean leftover-files
+RefuseManualStart=no
+RefuseManualStop=yes
+ 
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/ideascube clean leftover-files
```

Comment:
You might need to add a `TimeoutStartSec=` here.

On an old Koombook with lots of files to remove, I could see this taking more than the 90 seconds systemd gives services by default before considering they timed out and killing them.

I'm not sure what would a reasonable value... Maybe `TimeoutStartSec=5min`? (Let's not go crazy with `TimeoutStartSec=infinity` 😜)